internal/services/databricks/databricks_workspace_resource.go

```diff
@@ -85,12 +85,24 @@ func resourceDatabricksWorkspace() *pluginsdk.Resource {
 			},
 
 			"managed_services_cmk_key_vault_key_id": {
+				Type:         pluginsdk.TypeString,
+				Optional:     true,
+				ValidateFunc: keyVaultValidate.KeyVaultChildID,
+			},
+
+			"managed_disk_cmk_key_vault_key_id": {
 				Type:         pluginsdk.TypeString,
 				Optional:     true,
 				ForceNew:     true,
 				ValidateFunc: keyVaultValidate.KeyVaultChildID,
 			},
 
+			"managed_disk_cmk_rotation_to_latest_version_enabled": {
+				Type:         pluginsdk.TypeBool,
+				Optional:     true,
+				RequiredWith: []string{"managed_disk_cmk_key_vault_key_id"},
+			},
+
```

Comment:
Could it make sense to put these into a block called `managed_disk_cmk` and make both arguments required within it?

Reply:
I could, although what would it add? Rotation is `false` by default, and the `RequiredWIth` here is ensuring that it can't be set without setting the former argument, otherwise it can safely be ommitted.

Reply:
At first glance `managed_disk_cmk_key_vault_key_id` looks quite similar to `managed_services_cmk_key_vault_key_id` so the motivation for the suggestion was purely optical and to prevent confusion, but that's also subjective. I'm fine for these to remain top level properties.

Reply:
That's pretty much the case why I left it as a top-level property, since the other one isn't a block either. Potentially for 4.0 we could make a block of `customer_managed_key` and put all related properties inside of it? // cc @alexott